src/com/github/andlyticsproject/ChartFragment.java

```diff
@@ -91,8 +91,8 @@ public ChartFragment() {
 	public void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 
-		getActivity().getActionBar().setDisplayHomeAsUpEnabled(true);
-		getActivity().getActionBar().setTitle(getTitle());
+		//TODO reconfigure chart titles
```

Comment:
What is the plan with this TODO? It would be good to keep the using the chart title as otherwise it isn't clear what the different graphs represent.